app/javascript/components/enrollment/steps/ExposureInformation.js

```diff
@@ -204,6 +221,56 @@ class ExposureInformation extends React.Component {
       });
   };
 
+  handleCohortChange = (common_exposure_cohort, common_exposure_cohort_index) => {
+    // TODO: prevent adding duplicate cohorts
+    this.setState(
+      state => {
+        let common_exposure_cohorts = state.current.common_exposure_cohorts;
+        // Need to compare with undefined because index value of 0 evaluates to false
+        if (common_exposure_cohort_index === undefined) {
+          // Add new cohort to table
+          common_exposure_cohorts = common_exposure_cohorts ? common_exposure_cohorts.concat([common_exposure_cohort]) : [common_exposure_cohort];
+        } else {
+          // Update existing cohort from table
+          common_exposure_cohorts[`${common_exposure_cohort_index}`] = common_exposure_cohort;
+        }
+        return {
+          current: { ...state.current, common_exposure_cohorts },
+          modified: { ...state.modified, common_exposure_cohorts },
+          showCommonExposureCohortModal: false,
+          common_exposure_cohort: null,
+          common_exposure_cohort_index: null,
+        };
+      },
+      () => {
+        this.props.setEnrollmentState({ ...this.state.modified });
+      }
+    );
+  };
+
+  handleCohortDelete = async index => {
+    const self = this;
+    if (await confirmDialog('Are you sure you want to delete this common exposure cohort for this monitoree?', { title: 'Delete Common Exposure Cohort' })) {
+      self.setState(
+        state => {
+          const common_exposure_cohorts = state.current.common_exposure_cohorts;
+          common_exposure_cohorts.splice(index, 1);
+          return {
+            current: { ...state.current, common_exposure_cohorts },
+            modified: { ...state.modified, common_exposure_cohorts },
+          };
+        },
+        () => {
+          self.props.setEnrollmentState({ ...this.state.modified });
+        }
+      );
+    }
+  };
+
+  toggleCommonExposureCohortModal = (showCommonExposureCohortModal, common_exposure_cohort, common_exposure_cohort_index) => {
```

Comment:
`common_exposure_cohort` is snake case because it's something that belongs to `patient` (just like `first_positive_lab` in `CaseInformation`, and `showCommonExposureCohortModal` is consistent with all the other `show...Modal` variables in other components